_components/accordion/accordion.md

```diff
@@ -0,0 +1,26 @@
+---
+category: Components
+component:
+  name: accordion
+  status: ready
+  package: usa-accordion
+  dependencies:
```

Comment:
What's the intention for `dependencies`? Don't see it used anywhere

Reply:
It's in the packages section. Not really used much now, but perhaps more in the future.